src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -189,3 +199,9 @@ export interface PaypalCommerceScriptParams  {
     intent?: 'capture' | 'authorize';
     components?: ComponentsScriptType;
 }
+
+export enum PaymentStrategyTypes {
```

Comment:
Might be better to namespace it to avoid confusion, i.e.: `PaypalCommercePaymentStrategyType`.